o-fish-ios/Views/ReportFlow/Vessel/EMSInputView.swift

```diff
@@ -9,6 +9,7 @@ import SwiftUI
 
 struct EMSInputView: View {
     @ObservedObject var ems: EMSViewModel
+    @ObservedObject var attachments: AttachmentsViewModel
```

Comment:
This problem isn't limited to just this view - it's anywhere that a note is attached. Would be good to find a solution that's in `AddAttachmentsButton` or lower